config/bundles.php

```diff
@@ -9,4 +9,8 @@
     Doctrine\Bundle\DoctrineBundle\DoctrineBundle::class => ['all' => true],
     Doctrine\Bundle\MigrationsBundle\DoctrineMigrationsBundle::class => ['all' => true],
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'test' => true],
+    Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle::class => ['all' => true],
+    Symfony\Bundle\WebProfilerBundle\WebProfilerBundle::class => ['dev' => true, 'test' => true],
+    Symfony\Bundle\MonologBundle\MonologBundle::class => ['all' => true],
```

Comment:
Won't this fail in the `prod` environment when packages are installed without `require-dev` dependencies?

Reply:
Yes, probably. Unfortunately this is the out of the box-configuration from the debug-pack, which is otherwise only intended for dev/test. I should also require logger for all environments to fix this.